server/src/app.py

```diff
@@ -87,6 +87,12 @@ def startup_event() -> None:
 
         args = SimpleNamespace(**vars(CONFIG.admin_database))
```

Comment:
![image](https://github.com/dscc-admin-ch/lomas/assets/107670061/627476bd-aa4a-4323-bcee-d54ca8fc217d)

Actually, **vars(CONFIG.admin_database) works perfectly, I just missed it.

The reason for why it did not work is that the add_datasets function still used self.db instead of db in the available image. This did not raise any exception because self.db is a pymongo Collection object, and all methods used also exist for this class, they just don't have any effect on the database.